### ISSUE TITLE
Move to Sesame 4.0.0

### ISSUE THREAD

**Issue by TimmyStorms** [ontop/ontop#68]
Keep up with the latest Sesame releases. 2.7.10 dates back to January 2014. Keeping this old dependency makes it hard to use Ontop with other RDF frameworks who have made the switch.

Java 8 is required though for version 4.0.0, so this might be a deal breaker.

**Comment by ghxiao** [comment]
@TimmyStorms Indeed we would like to keep all the dependencies up-to-date ! 

We are in a large research project and we need to sync with all the other parters in the project for updating dependencies.
   
We have already decided to move to Java 8 soon. Sesame 4.0.0 might come after Java 8. Please be a bit patient :)

Meanwhile, if can tell us more about how you want to use Ontop with other RDF Framework, we will be super happy.

**Comment by ghxiao** [comment]
Started tracking in the branch feature/sesame4

**Comment by TimmyStorms** [comment]
Hi guys, any update on this? There hasn't been any development on the feature branch for over 2 months. Would love to see Sesame 4 support in there. Is there a date you're working towards, or has this low priority?

We're working with Stardog and would like to make the move from version 3.x to 4.x. As 4.x works with Sesame 4, we're having classpath issues with both sesame verions on there.

**Comment by ghxiao** [comment]
Hi @TimmyStorms , 

Unfortunately your impression it is right. We did not work on sesame v4 recently. The major reason is that we are bound to a large EU project.  Since the migration of sesame from v2 to v4 will be a major effort and break all partners' code, we need to coordinate the dependencies among all partners. Sorry that I could not promise a date now.

I am fully aware of the whole situation. Therefore I created issue #84.
